store/tikv/txn_test.go

```diff
@@ -0,0 +1,93 @@
+package tikv
+
+import (
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/util/testleak"
+)
+
+type testTxnSuite struct {
+	store *tikvStore
+}
+
+var _ = Suite(&testTxnSuite{})
+
+func (s *testTxnSuite) SetUpTest(c *C) {
+	s.store = newTestStore(c)
+}
+
+func (s *testTxnSuite) TearDownTest(c *C) {
+	s.store.Close()
+}
+
+func (s *testTxnSuite) TestSetCapAndReset(c *C) {
+	txn, err := newTiKVTxn(s.store)
+	c.Assert(err, IsNil)
+	txn.SetCap(100)
+	txn.Reset()
+	c.Assert(txn.us.Size(), Equals, 0)
+}
+
+func (s *testTxnSuite) TestString(c *C) {
+	txn, err := newTikvTxnWithStartTS(s.store, 1)
+	c.Assert(err, IsNil)
+	c.Assert(txn.String(), Equals, "1")
+}
+
+func (s *testTxnSuite) TestSeekReverse(c *C) {
+	defer testleak.AfterTest(c)()
+
+	txn, err := newTiKVTxn(s.store)
+	c.Assert(err, IsNil)
+	iter, err := txn.SeekReverse(nil)
```

Comment:
It test nothing

Reply:
It test whether it will report error if the input is nil.